scipy/signal/fir_filter_design.py

```diff
@@ -1,14 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
Noticed this after seeing the fancy comments with ``\pi`` and ``\integral``. We only have UTF-8 source code in a few recently added files. I'm not aware of any issues that may give (except for non-displaying characters in dumb editors), but adding a comment here just in case.